insights/parsers/readlink_openshift_certs.py

```diff
@@ -0,0 +1,73 @@
+"""
+Get the real path of the Openshift certificate symbolic link file
+=================================================================
+
+This module contains the following parsers:
+
+ReadLinkEKubeletClientCurrent - command ``/usr/bin/readlink -e /etc/origin/node/certificates/kubelet-client-current.pem``
+-------------------------------------------------------------------------------------------------------------------------
+ReadLinkEKubeletServerCurrent - command ``/usr/bin/readlink -e /etc/origin/node/certificates/kubelet-server-current.pem``
+-------------------------------------------------------------------------------------------------------------------------
+"""
+
+from insights.specs import Specs
+from insights.parsers import SkipException
+from insights import parser, CommandParser
+
+
+@parser(Specs.readlink_e_shift_cert_client)
+class ReadLinkEKubeletClientCurrent(CommandParser):
+    """
+    Class for command: /usr/bin/readlink -e /etc/origin/node/certificates/kubelet-client-current.pem
+
+    Sample content from command is::
+
+        /etc/origin/node/certificates/kubelet-client-2019-10-18-23-17-35.pem
+
+    Examples:
+        >>> client.path
+        '/etc/origin/node/certificates/kubelet-client-2019-10-18-23-17-35.pem'
+
+    Raises:
+        SkipException: When input content is empty
+    """
+    def parse_content(self, content):
+        if content is None or len(content) == 0:
+            raise SkipException("No Data from command: /usr/bin/readlink -e /etc/origin/node/certificates/kubelet-client-current.pem")
+
+        for line in content:
+            self._path = line
```

Comment:
@shlao this will set `self._path` to the last line in `content` so why not just use `self._path = content[-1]` instead of the for loop?